themes/default/content/docs/_index.md

```diff
@@ -11,7 +11,7 @@ h1: Pulumi Docs
 description: <p>Pulumi is an <a href="https://github.com/pulumi/pulumi" target="_blank">open source</a> infrastructure as code tool for creating, deploying, and managing cloud infrastructure.</p>
 link_buttons:
   primary:
-    label: Quickstart
+    label: Get Started
```

Comment:
Included this fix also -- we momentarily changed the name of the getting-started guide to Quickstart, but when we changed it back, we never went back and updated this button label.